### PR TITLE
Update default versions in recommender high and low deployments

### PR DESCRIPTION
**Pull request by jbartosik**
Missed in #5462
#5355

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ The current default version is Vertical Pod Autoscaler 0.12.0
 
 | VPA version     | Kubernetes version |
 |-----------------|--------------------|
+| 0.13            | 1.25+              |
```
Comment (kgolab) [inline vertical-pod-autoscaler/README.md:57]: If you're changing documentation as well - shouldn't the text above ("current default version") be also changed?
But maybe doc change should be a separate thing (e.g. part of Release Notes PR)?

**Comment by kgolab** [comment]
Thanks!

/lgtm

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (jbartosik) [reply vertical-pod-autoscaler/README.md:57]: Done

**Pull request merged**